todo_app/templates/index.html

```diff
@@ -20,9 +20,31 @@ <h1 class="display-4">To-Do App</h1>
 <br>
   <div class="row">
     <div class="col-md-12">
-      <h2>Items</h2>
+      <h2>To Do:</h2>
       <ul class="list-group mb-4">
-        {% for items in my_items %}
+        {% for items in view_model.todo_items %}
+          <li> {{ items.name }} - {{ items.status }}</li>
+          {% if items.status != "Done"%}
```

Comment:
We can drop these `if` checks now, since we know that everything in this list is in the to-do status by default!